LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2012-2014, Walmart and other contributors.
+Copyright (c) 2012-2017, Walmart and other contributors.
```

Comment:
Split it into two line:

```
Copyright (c) 2012-2017, Project contributors.
Copyright (c) 2012-2014, Walmart.
```

Since Walmart is no longer a contributor it should not get credit beyond 2014.